README.md

```diff
@@ -247,7 +247,7 @@ git push -u origin <your_branch_name>
l Code Fetcher](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/DialCodeFetcher.html)| 104. | [Dino Game](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/dino-game.html)| 105. | [Drawing App](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/drawingapp.html)| 106. | [Email Validator](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/EmailValidator.html)| 107. | [Find Num](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/findnum.html)| 108. | [Food Website](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/food_website.html)| 108. | [Form Validation](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/FormValidation.html)| 109. | [Login Form](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/loginform.html)| 110. | [Maps](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/maps.html)| 111. | [Mapty](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/mapty.html)| 112. | [Movie Quote Generator](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/MovieQuoteGenerator.html)| 113. | [Multiple Portfolio](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/multiple-portfolio.html)| 115. | [Music Player](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/musicPlayer.html)| 116. | [Palindrome](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/palindrome.html)| 117. | [Placeholder](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/placeholder.html)| 118. | [Quiz App](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/Quiz_app.html)| 119. | [Report Calculator](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/ReportCalculator.html)| 120. | [Responsive Landing](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/responsiveLanding.html)| 121. | [Reviews](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/reviews.html)| 122. | [Snake](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/snake.html)| 123. | [Socials](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/socials.html)| 124. | [Star Catching Game](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/star-catching-game.html)| 125. | [Sticky Notes](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/sticky_notes.html)| 126. | [Stopwatch](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/stopwatch.html)| 127. | [Tap Music](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/tap_music.html)| 128. | [Tiktok Clone](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/Tiktok_Clone.html)| 129. | [Toast Notification](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/toast_notification.html)| 130. | [To Do](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/todo.html)
```

Comment:
Your changes are not properly reflected. You can see the whole new addition appears in just line 250. Please change lines after every 3 addition.

Reply:
Made the required changes, @Astrodevil hope it's in the correct format now.